tslint.json

```diff
@@ -21,7 +21,8 @@
   },
   "linterOptions": {
     "exclude": [
-        "lib/lndclient/lndrpc_pb.d.ts"
+        "lib/lndclient/lndrpc_pb.d.ts",
+        "package.json"
```

Comment:
I don't think we need to exclude package.json here any longer since we're not using the `resolveJsonModule` compiler option, right?

Reply:
Yes you are right.